datafusion/src/optimizer/limit_push_down.rs

```diff
@@ -43,25 +42,6 @@ fn limit_push_down(
     execution_props: &ExecutionProps,
 ) -> Result<LogicalPlan> {
     match (plan, upper_limit) {
-        (
-            LogicalPlan::Explain {
-                verbose,
-                schema,
-                plan,
-                stringified_plans,
-            },
-            _,
-        ) => {
-            let schema = schema.as_ref().to_owned().into();
-            optimize_explain(
```

Comment:
this is the old style "explain" implementation that needed to be done for each optimizer for it to correctly show explain plans (and it was missing from several)